driver-scala/src/integration/scala/org/mongodb/scala/TestMongoClientHelper.scala

```diff
@@ -29,7 +29,7 @@ object TestMongoClientHelper {
 
   val mongoClientURI: String = {
     val uri = Properties.propOrElse(MONGODB_URI_SYSTEM_PROPERTY_NAME, DEFAULT_URI)
-    if (!uri.isBlank) uri else DEFAULT_URI
+    if (!uri.codePoints().allMatch((cp: Int) => Character.isWhitespace(cp))) uri else DEFAULT_URI
```

Comment:
`String.isBlank` was added in Java SE 11, so when I run `sdk use java 17.0.10-librca && ./gradlew -PjavaVersion=8 clean :driver-scala:integrationTest` locally, it fails.

The are two reasons this hasn't been a problem in Evergreen:

1. `run-tests.sh` did not run the `integrationTest` task (the `test` task does not depend on `integrationTest`).
2. `run-scala-tests.sh` did not use `JAVA_VERSION`. So, despite this script running `integrationTest` implicitly, it was using the same JDK Gradle is run by.